pkg/service/model_build_load_balancer.go

```diff
@@ -173,6 +179,22 @@ func (t *defaultModelBuildTask) buildLoadBalancerIPAddressType(_ context.Context
 	}
 }
 
+func (t *defaultModelBuildTask) buildSecurityGroupsInboundRulesOnPrivateLink(_ context.Context) (elbv2model.SecurityGroupsInboundRulesOnPrivateLinkStatus, error) {
+	securityGroupsInboundRulesOnPrivateLink := ""
+	if exists := t.annotationParser.ParseStringAnnotation(annotations.SvcLBSuffixEnforceSGInboundRulesOnPrivateLinkTraffic, &securityGroupsInboundRulesOnPrivateLink, t.service.Annotations); !exists {
+		return "", nil
+	}
+
+	switch securityGroupsInboundRulesOnPrivateLink {
+	case string(elbv2model.SecurityGroupsInboundRulesOnPrivateLinkOn):
+		return elbv2model.SecurityGroupsInboundRulesOnPrivateLinkOn, nil
+	case string(elbv2model.SecurityGroupsInboundRulesOnPrivateLinkOff):
+		return elbv2model.SecurityGroupsInboundRulesOnPrivateLinkOff, nil
+	default:
+		return "", errors.Errorf("unknown securityGroupsInboundRulesOnPrivateLink status: %v", securityGroupsInboundRulesOnPrivateLink)
```

Comment:
We should modify this error message to give them suggestion to use the allowed values as well. Something like ``errors.Errorf("Invalid value for securityGroupsInboundRulesOnPrivateLink status: %v, value must be one of [%v, %v]", securityGroupsInboundRulesOnPrivateLink), string(elbv2model.SecurityGroupsInboundRulesOnPrivateLinkOn), string(elbv2model.SecurityGroupsInboundRulesOnPrivateLinkOff)``